lib/preprocessor.js

```diff
@@ -36,7 +36,7 @@ var createNextProcessor = function (preprocessors, file, done) {
   }
 }
 
-var createPreprocessor = function (config, basePath, injector) {
+var createPreprocessor = function (config, basePath, ppOrder, injector) {
```

Comment:
I guess so.  We could have a custom property of `config` like `!!preprocessorOrder` which would never be needed to match a real file.